### ISSUE TITLE
detect if DNS is static

### ISSUE THREAD

**Issue by PipeItToDevNull** [PipeItToDevNull/Get-Specs#31]

**Comment by m6qh9mkyn9** [comment]
If you goto
Computer\HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\Tcpip\Parameters\Interfaces\
the DWORD EnableDHCP is present with either a value of 0 or 1 which will tell if you if the dns is static.

**Comment by PipeItToDevNull** [comment]
this is not true in my testing, also there is a key for each adapter which sucks 

BUT, `Nameserver` is only populated if DNS is static in my testing 

This cannot integrate with the standard registry checks already present due to the dynamic adapter naming
